### ISSUE TITLE
toggle_code block dont work

### ISSUE THREAD

**Issue by jidckii** [dauxio/daux.io#190]
**Describe the bug**
The block is not the same as in the documentation. 
And it doesn't work.
example: https://docs.yuccastream.com/en/install.html
I use docker image daux/daux.io:latest for build static files

**Screenshots**
![image](https://user-images.githubusercontent.com/6608896/74004401-7892d700-4997-11ea-8665-9a659dd340ec.png)
**Config**
```
{
  "title": "docs.yuccastream.com",
  "tagline": "Yucca docs",
  "author": "YuccaStream developers team",
  "image": "yucca.png",
  "ignore": {
    "files": [
      "README.md",
      ".gitlab-ci.yml",
      "Makefile",
      "AREAS",
      "CODEOWNERS"
    ],
    "folders": [
      "static"
    ]
  },
  "live": {
    "clean_urls": true
  },
  "html": {
    "auto_landing": true,
    "theme": "daux-blue",
    "breadcrumbs": true,
    "breadcrumb_separator": "Chevrons",
    "toggle_code": true,
    "float": true,
    "date_modified": true,
    "inherit_index": true,
    "links": {
      "yuccastream.com": "https://yuccastream.com"
    },
    "powered_by": "Powered by Daux.io"
  },
  "timezone": "Asia/Yekaterinburg",
  "languages": {
    "ru": "Russian",
    "en": "English"
  }
}

```

**Comment by onigoetz** [comment]
Hello, I can confirm this is a bug.
I will look into it.

**Comment by onigoetz** [comment]
This should be fixed now, can you try using the latest dev-master (or :latest in docker) ?

**Comment by jidckii** [comment]
![Screenshot_2020-02-08-23-19-18-890_com android chrome](https://user-images.githubusercontent.com/6608896/74090122-d32c5000-4ac9-11ea-90b1-282f16fbf264.jpg)
You remove key `--search` ?

**Comment by jidckii** [comment]
I removed key `--search` and build successful.
Now the checkbox works )
But, i see simple checkbox 
![image](https://user-images.githubusercontent.com/6608896/74097279-2931df80-4b2c-11ea-9b9f-287be9e1b0ca.png)
and I expect to see the menu, as you have on the site
![image](https://user-images.githubusercontent.com/6608896/74097310-6c8c4e00-4b2c-11ea-8890-d7cf724b8f04.png)

**Comment by onigoetz** [comment]
Oh sorry, I removed this menu in version 0.12 (https://github.com/dauxio/daux.io/commit/ef496e9381e0dee76d8fe729bb0b18a06ad8f5e1#diff-4b8648c180a014fca8f19ecaf9c552d7) :/ I forgot to update the site to the latest version.

To be honest I even hesitated to remove the feature entirely, a documentation without the code blocks seems a bit useless.

**Comment by onigoetz** [comment]
Fix available in version [0.12.1](https://github.com/dauxio/daux.io/releases/tag/0.12.1)

**Comment by onigoetz** [comment]
I also updated the documentation to clear the confusion on what the documentation looks like after generation
